src/screens/Event/EventAdd.js

```diff
@@ -458,7 +477,7 @@ type Props = {
               <TouchableOpacity
                 onPress={() => this.setState({ isLocationPickerVisible: !this.state.isLocationPickerVisible })}
               >
-                <Value>{address ? formatted[0] : 'Set a location'}</Value>
+                <Value>{location ? formatted.split('-')[0] : 'Set a location'}</Value>
```

Comment:
is it still necessary to check `location`?

Reply:
Yes, 'cause there's two possibilites for the 'location' field: something from the server (user is editing the event) or something that the user is currently typing.